wso2is_analytics/manifests/params.pp

```diff
@@ -160,14 +161,18 @@
       }
     }
 
+    $analytics_event_store_datasource = 'wso2_analytics_event_store_db'
```

Comment:
why the key refers to a data source and value refers to a database ?

Reply:
That's the default configuration. All product default configurations have 'db's.
If we are to change this point we need to change all IMO